packages/cxl-ui/src/components/cxl-marketing-nav.js

```diff
@@ -233,13 +233,29 @@ export class CXLMarketingNavElement extends LitElement {
    * @see https://app.clickup.com/t/3rgnekt
    * @private
    */
-  _onOverlayOpen(oe) {
-    const overlay = oe.target;
+  _onOverlayOpen(e) {
+    const overlay = e.target;
 
+    // Is this needed?
     if (!overlay) {
       return;
     }
 
+    // Check if the child overlay is larger than the top level overlay.
+    const overlays = document.querySelectorAll('vaadin-context-menu-overlay');
```

Comment:
This is always guaranteed to return the right thing? There's no way to scope this query more accurately that `document`?

Reply:
Since all overlays are added to the `document.body`, I'm not sure there is any benefit to using:

```
const overlays = document.querySelectorAll('body > vaadin-context-menu-overlay');
```

Currently _all_ context menus would be targeted by these methods, though this shouldn't be an issue as we would most likely want this behavior on all context menus.

Reply:
If there's significant concern on specificity, this is where this issue would come in to play: https://cxlworld.slack.com/archives/C01JABH8AHX/p1674053401244969

Declaratively we would target only `cxl-marketing-nav` though this would require a refactor.

Reply:
Eh, I just looked in inspector, and all overlays have usable identificaton like `document.querySelectorAll('vaadin-context-menu-overlay[theme="cxl-marketing-nav"]`.